developer/images/dependencies-update/tasks/update_kcp_version.sh

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 The Pipeline Service Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+usage() {
+  echo "
+Usage:
+    ${0##*/} [options]
+
+Update the kcp version.
+
+Optional arguments:
+    -d, --debug
+        Activate tracing/debug mode.
+    -h, --help
+        Display this message.
+
+Example:
+    ${0##*/}
+" >&2
+}
+
+parse_args() {
+  while [[ $# -gt 0 ]]; do
+    case $1 in
+    -d | --debug)
+      set -x
+      DEBUG="--debug"
+      export DEBUG
+      ;;
+    -h | --help)
+      usage
+      exit 0
+      ;;
+    *)
+      echo "Unknown argument: $1"
+      usage
+      exit 1
+      ;;
+    esac
+    shift
+  done
+}
+
+init() {
+  CONFIG="$PROJECT_DIR/developer/ckcp/config.yaml"
+}
+
+get_current_version() {
+  current_kcp_version="$(yq '.kcp.version' "$CONFIG" | sed 's/v//')"
+  if [ -z "$current_kcp_version" ]; then
+    printf "[ERROR] Could not retrieve the current kcp version\n" >&2
+    exit 1
+  fi
+}
+
+get_latest_version() {
+  latest_kcp_version="$(curl -s https://api.github.com/repos/kcp-dev/kcp/releases/latest | yq '.tag_name' | sed 's/v//')"
+  if [ -z "$latest_kcp_version" ]; then
+    printf "[ERROR] Could not retrieve the latest kcp version\n" >&2
+    exit 1
+  fi
+}
+
+update_kcp_version() {
+  printf "Upgrade kcp version from '%s' to '%s'\n" "$current_kcp_version" "$latest_kcp_version" | tee "$COMMIT_MSG"
+  sed -i "s,$current_kcp_version,$latest_kcp_version,g" .github/workflows/build-push-images.yaml
```

Comment:
I verified it in my Linux VM and it reported errror
>sed: can't read .github/workflows/build-push-images.yaml: No such file or directory

Reply:
Should be fixed now, please retry. Make sure to call it via `update.sh --task update_kcp_version`, as it will set a required env var.